Makefile.toml

```diff
@@ -1,28 +1,31 @@
+[env]
+LOCALAUTH0_CONFIG_PATH = "/home/app/localauth0.toml"
```

Comment:
I think this file should be retrieved from outside the container

Reply:
So `./localhost.toml`? Or maybe even `localhost.toml` if it works.